toast-tk-plugin-loader/src/main/java/io/toast/tk/plugin/PluginLoader.java

```diff
@@ -0,0 +1,92 @@
+package io.toast.tk.plugin;
+
+import java.io.File;
+import java.io.FileFilter;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.FileFilter'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)